analytics.js

```diff
@@ -872,6 +872,11 @@ class Analytics {
       });
       throw Error("failed to initialize");
     }
+    if (options && options.setCookieDomain) {
+      this.storage.setCookieDomain(options.setCookieDomain);
```

Comment:
Since I don't know everything about this context so please test this line and make sure its working or else we can discuss.

Reply:
yes tested this..